mds/authent/migrations/0003_auto_20190517_1158.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.7 on 2019-05-17 11:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Don't know why I have this migration

Reply:
Well, must have missed it at some point. You could do that in a separated commit

Reply:
Yes, I've moved it to another PR